src/jwt.cc

```diff
@@ -24,16 +24,16 @@
 namespace google {
 namespace jwt_verify {
 
-namespace {
-
-static const absl::flat_hash_set<std::string> kImplementedAlgs = {
+bool hasImplementedAlg(std::string alg) {
+  static const absl::flat_hash_set<std::string> implemented_algs = {
```

Comment:
call you use "new" to create it

Reply:
If I use new, wouldn't the `flat_hash_set` leak? Do you mean

```
bool isImplemented(absl::string_view alg) {
  static const absl::flat_hash_set<absl::string_view> *implemented_algs =
      new absl::flat_hash_set<absl::string_view>({
          {"ES256"}, {"ES384"}, {"ES512"},
          {"HS256"}, {"HS384"}, {"HS512"},
          {"RS256"}, {"RS384"}, {"RS512"},
          {"PS256"}, {"PS384"}, {"PS512"},
          {"EdDSA"},
        });

  return implemented_algs->find(alg) != implemented_algs->end();
}
```

Reply:
Oh I see this http://www.cs.technion.ac.il/users/yechiel/c++-faq/construct-on-first-use-v2.html

Reply:
Okay, here 1015e94!